Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -298,10 +219,8 @@ class BRBLoginViewController: UIViewController {
         netidTextField.resignFirstResponder()
         passwordTextField.resignFirstResponder()
         
-        if BRBAccountSettings.saveLoginInfo {
-            let loginInfo = (netid: netid, password: password)
-            BRBAccountSettings.saveToKeychain(loginInfo: loginInfo)
-        }
+        let loginInfo = (netid: netid, password: password)
+        accountManager?.saveLoginInfo(loginInfo: loginInfo)
```

Comment:
We can get rid of the `accountManager` from the `LoginViewController` by just having the delegate of the `LoginViewController` (i.e. the `BRBViewController`) save the login info automatically whenever the user logs in.